pkg/app/managedisc/factory.go

```diff
@@ -48,24 +42,23 @@ func (f *Factory) VisorUpdater(port uint16) Updater {
 		DiscAddr: f.ServiceDisc,
 	}
 
-	return &serviceUpdater{
-		client:   servicedisc.NewClient(f.Log, conf),
-		interval: f.UpdateInterval,
+	return &serviceManager{
+		client: servicedisc.NewClient(f.Log, conf),
 	}
 }
 
-// AppUpdater obtains an app updater based on the app name and configuration.
-func (f *Factory) AppUpdater(conf appcommon.ProcConfig) (Updater, bool) {
-	// Always return empty updater if keys are not set.
+// AppManager obtains an app manager based on the app name and configuration.
```

Comment:
Similarly to above, this does not actually manage apps, but updates entries of apps in a discovery.

Reply:
Done.